stdlib/xml/dom/minidom.pyi

```diff
@@ -13,15 +15,19 @@ def parse(
     file: str | SupportsRead[ReadableBuffer | str], parser: XMLReader | None = None, bufsize: int | None = None
 ) -> Document: ...
 def parseString(string: str | ReadableBuffer, parser: XMLReader | None = None) -> Document: ...
-def getDOMImplementation(features=None) -> DOMImplementation | None: ...
+def getDOMImplementation(features: str | Iterable[tuple[str, str | None]] | None = None) -> DOMImplementation | None: ...
 
 class Node(xml.dom.Node):
+    parentNode: Node | None
+    childNodes: NodeList
```

Comment:
To silence pyright. Need to import `Incomplete` from `_typeshed`.
```suggestion
    childNodes: NodeList[Incomplete]
```

(same for the `NodeList` annotations below)